frontend/src/app/components/header/header.component.ts

```diff
@@ -15,18 +16,18 @@ export class HeaderComponent implements OnInit, OnDestroy {
   public logoPicture: string = "assets/img/logo.png";
   public logInButtonTitle: string = ButtonLabel.LOG_IN;
   public isActiveButtonMenu: boolean = false;
-  public isActiveButtonsAuth: boolean = true;
   private subscription: Subscription;
+  public editButtonTitle: string = ButtonLabel.CONTENT_MANAGEMENT;
+  public role: string = '';
+  public readonly adminRole: string = AuthRole.ADMIN;
```

Comment:
It is not a string. You set AuthRole type but defined type of property a string. It is wrong